lte/gateway/python/magma/pipelined/app/uplink_bridge.py

```diff
@@ -168,7 +179,7 @@ def _delete_all_flows(self):
     def _install_flow(self, priority: int, flow_match: str, flow_action: str):
         if self.config.enable_nat is True:
             return
-        flow_cmd = "ovs-ofctl add-flow %s \"priority=%s,%s, actions=%s\"" % (
+        flow_cmd = "ovs-ofctl add-flow -Oopenflow13 %s \"priority=%s,%s, actions=%s\"" % (
```

Comment:
missing space should be `-O OpenFlow13`

Reply:
space is not required, infact, most of time I have seen no spaces in ovs code base.